internal/lookout/configuration/types.go

```diff
@@ -22,6 +22,9 @@ type LookoutUIConfig struct {
 	OverviewAutoRefreshMs int
 	JobSetsAutoRefreshMs  int
 	JobsAutoRefreshMs     int
+
+	FakeDataEnabled     bool
```

Comment:
Not sure why you'd need this on the server-side config, since it's only enabled via a client-side query param?